velox/expression/EvalCtx.h

```diff
@@ -36,7 +37,11 @@ class PeeledEncoding;
 // flags for Expr interpreter.
 class EvalCtx {
  public:
-  EvalCtx(core::ExecCtx* execCtx, ExprSet* exprSet, const RowVector* row);
+  EvalCtx(
+      core::ExecCtx* execCtx,
+      ExprSet* exprSet,
+      const RowVector* row,
+      DriverCtx* driverCtx = nullptr);
```

Comment:
How much do you need to change if we remove the default nullptr?

Reply:
hey @Yuhta  total need modify additional more 32 files .

![image](https://github.com/facebookincubator/velox/assets/9858245/8f2f62c3-6f44-43cf-b278-2c8a3821e464)